docs/rst/installation/linux_binary_installation.rst

```diff
@@ -3,12 +3,55 @@
 Linux binary installation
 =========================
 
-Debian packages for Vulcanexus Galactic Gamble are currently available for Ubuntu Focal.
-Please, start following the `Installation Guide for ROS 2 Galactic Geochelone <https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Debians.html>`_ setting the locale to support ``UTF-8`` (if necessary) and the ROS 2 apt repository.
-Instead of proceeding with installing the ROS 2 packages, follow the steps below.
+Debian packages for Vulcanexus Galactic Gamble are currently available for Ubuntu Focal. Since Vulcanexus is a ROS 2 overlay, certain ROS 2 prerequisites need to be met before installing.
 
-Setup sources
--------------
+ROS 2 prerequisites
+-------------------
+
+First of all, setup a UTF-8 locale as required by ROS 2. Locale settings can be checked and set up with the following commands:
+
+.. code-block:: bash
+
+    locale  # check for UTF-8
+
+    sudo apt update && sudo apt install locales
+    sudo locale-gen en_US en_US.UTF-8
+    sudo update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
+    export LANG=en_US.UTF-8
```

Comment:
I think we should indicate this is an example and that any other `UTF-8` locale is also valid. This one is the one used when testing ROS 2, but it is not strictly required.